erlang-25.yaml

```diff
@@ -71,3 +72,19 @@ update:
     strip-prefix: OTP-
     use-tag: true
     tag-filter: OTP-25
+
+test:
+  pipeline:
+    - runs: |
+        erl -eval 'erlang:display(erlang:system_info(otp_release)), halt().'  -noshell
+    - runs: |
+        cat <<EOF > "hello.erl"
```

Comment:
Can you add a brief comment on what this test is doing and why?

Reply:
Done